### ISSUE TITLE
[Bug]: efficientAD trianing break

### ISSUE THREAD

**Issue by monument-and-sea-all-the-gift** [openvinotoolkit/anomalib#1288]
### Describe the bug

Traceback (most recent call last):
  File "tools/train.py", line 79, in <module>
    train(args)
  File "tools/train.py", line 64, in train
    trainer.fit(model=model, datamodule=datamodule)
  File "/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/pytorch_lightning/trainer/trainer.py", line 608, in fit
    call._call_and_handle_interrupt(
  File "/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/pytorch_lightning/trainer/call.py", line 38, in _call_and_handle_interrupt
    return trainer_fn(*args, **kwargs)
  File "/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/pytorch_lightning/trainer/trainer.py", line 650, in _fit_impl
    self._run(model, ckpt_path=self.ckpt_path)
  File "/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/pytorch_lightning/trainer/trainer.py", line 1112, in _run
    results = self._run_stage()
  File "/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/pytorch_lightning/trainer/trainer.py", line 1191, in _run_stage
    self._run_train()
  File "/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/pytorch_lightning/trainer/trainer.py", line 1214, in _run_train
    self.fit_loop.run()
  File "/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/pytorch_lightning/loops/loop.py", line 199, in run
    self.advance(*args, **kwargs)
  File "/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/pytorch_lightning/loops/fit_loop.py", line 267, in advance
    self._outputs = self.epoch_loop.run(self._data_fetcher)
  File "/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/pytorch_lightning/loops/loop.py", line 200, in run
    self.on_advance_end()
  File "/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/pytorch_lightning/loops/epoch/training_epoch_loop.py", line 250, in on_advance_end
    self._run_validation()
  File "/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/pytorch_lightning/loops/epoch/training_epoch_loop.py", line 308, in _run_validation
    self.val_loop.run()
  File "/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/pytorch_lightning/loops/loop.py", line 194, in run
    self.on_run_start(*args, **kwargs)
  File "/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/pytorch_lightning/loops/dataloader/evaluation_loop.py", line 132, in on_run_start
    self._on_evaluation_start()
  File "/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/pytorch_lightning/loops/dataloader/evaluation_loop.py", line 246, in _on_evaluation_start
    self.trainer._call_lightning_module_hook(hook_name, *args, **kwargs)
  File "/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/pytorch_lightning/trainer/trainer.py", line 1356, in _call_lightning_module_hook
    output = fn(*args, **kwargs)
  File "/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/anomalib-0.6.0-py3.8.egg/anomalib/models/efficientad/lightning_model.py", line 231, in on_validation_start
  File "/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/torch/utils/_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
  File "/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/anomalib-0.6.0-py3.8.egg/anomalib/models/efficientad/lightning_model.py", line 175, in map_norm_quantiles
RuntimeError: quantile() input tensor is too large
Epoch 99:  53%|█████▎    | 448/844 [00:50<00:44,  8.93it/s, loss=12.5, train_st_step=4.720, train_ae_step=0.443, train_stae_step=0.0451, train_loss_step=5.210, image_F1Score=0.852, image_AUROC=0.732, train_st_epoch=4.480, train_ae_epoch=0.485, train_stae_epoch=0.0502, train_loss_epoch=5.020]


### Dataset

Other (please specify in the text field below)

### Model

Other (please specify in the field below)

### Steps to reproduce the behavior


training custom data
python tools/train.py --config ./src/anomalib/models/efficientad/config.yaml
when training  efficientAD in epoch 99，break

### OS information

OS information:
- OS: [e.g. Ubuntu 20.04]
- Python version: [e.g. 3.8.10]
- Anomalib version: [e.g. 0.3.6]
- PyTorch version: [e.g. 1.9.0]
- CUDA/cuDNN version: [e.g. 11.1]
- GPU models and configuration: [e.g. 2x GeForce RTX 3090]
- Any other relevant information: [e.g. I'm using a custom dataset]


### Expected behavior

when training  efficientAD in epoch 99，break

### Screenshots

_No response_

### Pip/GitHub

GitHub

### What version/branch did you use?

_No response_

### Configuration YAML

```yaml
dataset:
  name: bpwkfl2
  format: folder
  path: /aseit-data/LKP_WorkSpace/data/hxqpsfl
  normal_dir: good # name of the folder containing normal images.
  abnormal_dir: broken
  mask_dir: null #optional
  normal_test_dir: null # optional
  extensions: null
  task: classification
  train_batch_size: 1
  eval_batch_size: 1
  num_workers: 1
  image_size: 256 # dimensions to which images are resized (mandatory)
  center_crop: null # dimensions to which images are center-cropped after resizing (optional)
  normalization: none # data distribution to which the images will be normalized: [none, imagenet]
  transform_config:
    train: null
    eval: null
  test_split_mode: from_dir # options: [from_dir, synthetic]
  test_split_ratio: 0.2 # fraction of train images held out testing (usage depends on test_split_mode)
  val_split_mode: same_as_test # options: [same_as_test, from_test, synthetic]
  val_split_ratio: 0.5 # fraction of train/test images held out for validation (usage depends on val_split_mode)

model:
  name: efficientad
  teacher_out_channels: 384
  model_size: medium # options: [small, medium]
  lr: 0.0001
  weight_decay: 0.00001
  padding: true
  # generic params
  normalization_method: min_max # options: [null, min_max, cdf]

metrics:
  image:
    - F1Score
    - AUROC
  pixel:
    - F1Score
    - AUROC
  threshold:
    method: adaptive #options: [adaptive, manual]
    manual_image: null
    manual_pixel: null

visualization:
  show_images: False # show images on the screen
  save_images: True # save images to the file system
  log_images: False # log images to the available loggers (if any)
  image_save_path: null # path to which images will be saved
  mode: full # options: ["full", "simple"]

project:
  seed: 42
  path: ./results

logging:
  logger: [] # options: [comet, tensorboard, wandb, csv] or combinations.
  log_graph: false # Logs the model graph to respective logger.

optimization:
  export_mode: null # options: torch, onnx, openvino
# PL Trainer Args. Don't add extra parameter here.
trainer:
  enable_checkpointing: true
  default_root_dir: null
  gradient_clip_val: 0
  gradient_clip_algorithm: norm
  num_nodes: 1
  devices: 1
  enable_progress_bar: true
  overfit_batches: 0.0
  track_grad_norm: -1
  check_val_every_n_epoch: 1
  fast_dev_run: false
  accumulate_grad_batches: 1
  max_epochs: 100
  min_epochs: null
  max_steps: -1
  min_steps: null
  max_time: null
  limit_train_batches: 1.0
  limit_val_batches: 1.0
  limit_test_batches: 1.0
  limit_predict_batches: 1.0
  val_check_interval: 1.0
  log_every_n_steps: 50
  accelerator: auto # <"cpu", "gpu", "tpu", "ipu", "hpu", "auto">
  strategy: null
  sync_batchnorm: false
  precision: 32
  enable_model_summary: true
  num_sanity_val_steps: 0
  profiler: null
  benchmark: false
  deterministic: false
  reload_dataloaders_every_n_epochs: 0
  auto_lr_find: false
  replace_sampler_ddp: true
  detect_anomaly: false
  auto_scale_batch_size: false
  plugins: null
  move_metrics_to_cpu: false
  multiple_trainloader_mode: max_size_cycle
```


### Logs

```shell
To use wandb logger install it using `pip install wandb`
/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/anomalib-0.6.0-py3.8.egg/anomalib/config/config.py:275: UserWarning: config.project.unique_dir is set to False. This does not ensure that your results will be written in an empty directory and you may overwrite files.
Global seed set to 42
2023-08-22 11:23:29,035 - anomalib.data - INFO - Loading the datamodule
2023-08-22 11:23:29,036 - anomalib.data.utils.transform - INFO - No config file has been provided. Using default transforms.
2023-08-22 11:23:29,036 - anomalib.data.utils.transform - INFO - No config file has been provided. Using default transforms.
2023-08-22 11:23:29,036 - anomalib.models - INFO - Loading the model.
2023-08-22 11:23:29,037 - anomalib.models.components.base.anomaly_module - INFO - Initializing EfficientadLightning model.
/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/torchmetrics/utilities/prints.py:36: UserWarning: Metric `PrecisionRecallCurve` will save all targets and predictions in buffer. For large datasets this may lead to large memory footprint.
  warnings.warn(*args, **kwargs)
2023-08-22 11:23:29,155 - anomalib.models.efficientad.lightning_model - INFO - Load pretrained teacher model from pre_trained/efficientad_pretrained_weights/pretrained_teacher_medium.pth
2023-08-22 11:23:29,227 - anomalib.utils.loggers - INFO - Loading the experiment logger(s)
2023-08-22 11:23:29,228 - anomalib.utils.callbacks - INFO - Loading the callbacks
/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/anomalib-0.6.0-py3.8.egg/anomalib/utils/callbacks/__init__.py:142: UserWarning: Export option: None not found. Defaulting to no model export
2023-08-22 11:23:29,283 - pytorch_lightning.utilities.rank_zero - INFO - GPU available: True (cuda), used: True
2023-08-22 11:23:29,283 - pytorch_lightning.utilities.rank_zero - INFO - TPU available: False, using: 0 TPU cores
2023-08-22 11:23:29,283 - pytorch_lightning.utilities.rank_zero - INFO - IPU available: False, using: 0 IPUs
2023-08-22 11:23:29,283 - pytorch_lightning.utilities.rank_zero - INFO - HPU available: False, using: 0 HPUs
2023-08-22 11:23:29,284 - pytorch_lightning.utilities.rank_zero - INFO - `Trainer(limit_train_batches=1.0)` was configured so 100% of the batches per epoch will be used..
2023-08-22 11:23:29,284 - pytorch_lightning.utilities.rank_zero - INFO - `Trainer(limit_val_batches=1.0)` was configured so 100% of the batches will be used..
2023-08-22 11:23:29,284 - pytorch_lightning.utilities.rank_zero - INFO - `Trainer(limit_test_batches=1.0)` was configured so 100% of the batches will be used..
2023-08-22 11:23:29,284 - pytorch_lightning.utilities.rank_zero - INFO - `Trainer(limit_predict_batches=1.0)` was configured so 100% of the batches will be used..
2023-08-22 11:23:29,284 - pytorch_lightning.utilities.rank_zero - INFO - `Trainer(val_check_interval=1.0)` was configured so validation will run at the end of the training epoch..
2023-08-22 11:23:29,284 - anomalib - INFO - Training the model.
2023-08-22 11:23:29,519 - pytorch_lightning.utilities.rank_zero - INFO - You are using a CUDA device ('NVIDIA GeForce RTX 3090') that has Tensor Cores. To properly utilize them, you should set `torch.set_float32_matmul_precision('medium' | 'high')` which will trade-off precision for performance. For more details, read https://pytorch.org/docs/stable/generated/torch.set_float32_matmul_precision.html#torch.set_float32_matmul_precision
2023-08-22 11:23:29,631 - anomalib.data.base.datamodule - INFO - No normal test images found. Sampling from training set using a split ratio of 0.20
2023-08-22 11:23:29,651 - anomalib.utils.callbacks.metrics_configuration - WARNING - Cannot perform pixel-level evaluation when task type is classification. Ignoring the following pixel-level metrics: ['F1Score', 'AUROC']
/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/torchmetrics/utilities/prints.py:36: UserWarning: Metric `ROC` will save all targets and predictions in buffer. For large datasets this may lead to large memory footprint.
  warnings.warn(*args, **kwargs)
2023-08-22 11:23:31,763 - pytorch_lightning.accelerators.cuda - INFO - LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0,1]
2023-08-22 11:23:31,766 - pytorch_lightning.callbacks.model_summary - INFO - 
  | Name                  | Type                     | Params
-------------------------------------------------------------------
0 | image_threshold       | AnomalyScoreThreshold    | 0     
1 | pixel_threshold       | AnomalyScoreThreshold    | 0     
2 | model                 | EfficientADModel         | 20.7 M
3 | image_metrics         | AnomalibMetricCollection | 0     
4 | pixel_metrics         | AnomalibMetricCollection | 0     
5 | normalization_metrics | MinMax                   | 0     
-------------------------------------------------------------------
20.7 M    Trainable params
0         Non-trainable params
20.7 M    Total params
82.957    Total estimated model params size (MB)
/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/pytorch_lightning/trainer/connectors/data_connector.py:224: PossibleUserWarning: The dataloader, train_dataloader, does not have many workers which may be a bottleneck. Consider increasing the value of the `num_workers` argument` (try 16 which is the number of cpus on this machine) in the `DataLoader` init to improve performance.
  rank_zero_warn(
/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/pytorch_lightning/trainer/connectors/data_connector.py:224: PossibleUserWarning: The dataloader, val_dataloader 0, does not have many workers which may be a bottleneck. Consider increasing the value of the `num_workers` argument` (try 16 which is the number of cpus on this machine) in the `DataLoader` init to improve performance.
  rank_zero_warn(
Training: 0it [00:00, ?it/s]2023-08-22 11:23:31,776 - anomalib.models.efficientad.lightning_model - INFO - Calculate teacher channel mean and std
Calculate teacher channel mean: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 448/448 [00:15<00:00, 29.30it/s]
Calculate teacher channel std: 100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 448/448 [00:00<00:00, 15018.45it/s]
Epoch 0:   0%|                                                                                                                                                                                                           | 0/844 [00:00<?, ?it/s]/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/pytorch_lightning/core/module.py:493: UserWarning: You called `self.log('train_st', ..., logger=True)` but have no logger configured. You can enable one by doing `Trainer(logger=ALogger(...))`
  rank_zero_warn(
/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/pytorch_lightning/core/module.py:493: UserWarning: You called `self.log('train_ae', ..., logger=True)` but have no logger configured. You can enable one by doing `Trainer(logger=ALogger(...))`
  rank_zero_warn(
/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/pytorch_lightning/core/module.py:493: UserWarning: You called `self.log('train_stae', ..., logger=True)` but have no logger configured. You can enable one by doing `Trainer(logger=ALogger(...))`
  rank_zero_warn(
/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/pytorch_lightning/core/module.py:493: UserWarning: You called `self.log('train_loss', ..., logger=True)` but have no logger configured. You can enable one by doing `Trainer(logger=ALogger(...))`
  rank_zero_warn(
Epoch 99:  53%|▌| 448/844 [00:2023-08-22 12:47:22,045 - anomalib.models.efficientad.lightning_model - INFO - Calculate Validation Dataset Quantiles10, image_F1Score=0.852, image_AUROC=0.732, train_st_epoch=4.480, train_ae_epoch=0.485, train_
Calculate Validation Dataset Quantiles: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 448/448 [00:07<00:00, 60.04it/s]
Traceback (most recent call last):
  File "tools/train.py", line 79, in <module>
    train(args)
  File "tools/train.py", line 64, in train
    trainer.fit(model=model, datamodule=datamodule)
  File "/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/pytorch_lightning/trainer/trainer.py", line 608, in fit
    call._call_and_handle_interrupt(
  File "/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/pytorch_lightning/trainer/call.py", line 38, in _call_and_handle_interrupt
    return trainer_fn(*args, **kwargs)
  File "/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/pytorch_lightning/trainer/trainer.py", line 650, in _fit_impl
    self._run(model, ckpt_path=self.ckpt_path)
  File "/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/pytorch_lightning/trainer/trainer.py", line 1112, in _run
    results = self._run_stage()
  File "/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/pytorch_lightning/trainer/trainer.py", line 1191, in _run_stage
    self._run_train()
  File "/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/pytorch_lightning/trainer/trainer.py", line 1214, in _run_train
    self.fit_loop.run()
  File "/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/pytorch_lightning/loops/loop.py", line 199, in run
    self.advance(*args, **kwargs)
  File "/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/pytorch_lightning/loops/fit_loop.py", line 267, in advance
    self._outputs = self.epoch_loop.run(self._data_fetcher)
  File "/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/pytorch_lightning/loops/loop.py", line 200, in run
    self.on_advance_end()
  File "/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/pytorch_lightning/loops/epoch/training_epoch_loop.py", line 250, in on_advance_end
    self._run_validation()
  File "/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/pytorch_lightning/loops/epoch/training_epoch_loop.py", line 308, in _run_validation
    self.val_loop.run()
  File "/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/pytorch_lightning/loops/loop.py", line 194, in run
    self.on_run_start(*args, **kwargs)
  File "/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/pytorch_lightning/loops/dataloader/evaluation_loop.py", line 132, in on_run_start
    self._on_evaluation_start()
  File "/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/pytorch_lightning/loops/dataloader/evaluation_loop.py", line 246, in _on_evaluation_start
    self.trainer._call_lightning_module_hook(hook_name, *args, **kwargs)
  File "/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/pytorch_lightning/trainer/trainer.py", line 1356, in _call_lightning_module_hook
    output = fn(*args, **kwargs)
  File "/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/anomalib-0.6.0-py3.8.egg/anomalib/models/efficientad/lightning_model.py", line 231, in on_validation_start
  File "/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/torch/utils/_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
  File "/home/aseit/anaconda3/envs/lkp/lib/python3.8/site-packages/anomalib-0.6.0-py3.8.egg/anomalib/models/efficientad/lightning_model.py", line 175, in map_norm_quantiles
RuntimeError: quantile() input tensor is too large
Epoch 99:  53%|█████▎    | 448/844 [00:50<00:44,  8.93it/s, loss=12.5, train_st_step=4.720, train_ae_step=0.443, train_stae_step=0.0451, train_loss_step=5.210, image_F1Score=0.852, image_AUROC=0.732, train_st_epoch=4.480, train_ae_epoch=0.485, train_stae_epoch=0.0502, train_loss_epoch=5.020]
```


### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by samet-akcay** [comment]
@monument-and-sea-all-the-gift, this seems to be a duplicate of #1177 and has been fixed in #1182. As @alexriedel1 pointed out, you could use the latest GitHub version of anomalib. These changes have not been pushed to pypi yet. We plan to release the new version to pypi this week.

I'm closing this issue for now Feel free to re-open is you still have some other issues.

**Comment by monument-and-sea-all-the-gift** [comment]
When will the bug be fixed and the new version topypi be released,thank you
